elasticsearch-odfe/plan.sh

```diff
@@ -0,0 +1,96 @@
+ELASTICSEARCH_VERSION="6.5.4"
+ELASTICSEARCH_PKG_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-$ELASTICSEARCH_VERSION.tar.gz"
```

Comment:
Is this source for how the opendistro builds it's own packages?  I did a quick scan an only saw pre-packaged versions available, no source tarballs.  I'd like to make sure we're pulling from the correct location.

Reply:
Yes, this is how it's built in opendistro. 👍